Tweaks

telemachus opened the pull request:
I apologize if levee is completely dormant for you and not something
you want pull requests for.

There are two basic things here:

1. The first just allows levee to compile on OS X. Currently there are
errors because of name clashes.
1. The second maps `g` in visual mode to be the equivalent of `1G`.
I think "go to start of file" is a pretty common need, and `g` is unused,
so why not?

Thanks for your time.

Orc commented:
Levee is pretty stable for me (the two major bugs -- file size and nonwrapping long lines -- were ones that I learned to live with a long time ago and have never been motivated enough to fix :-) so it's moving very slowly;  The OSX 10.5 binary I built in 2012 still runs on 10.10, so I missed `getline()` and `getcontext()` being absorbed into the g-dd-mn C standard of the week.

Thank you for the patch;   Can you redo the g = 1G as a hard macro (set 'g' to `HARDMACRO` in `globals.c`, add `'g'` --> `"1G"`  in `editcor.c::macrocommand()`, marvel at my software engineering skills from the early 1980s) before I merge your changes?

telemachus commented:
@Orc Thanks for the feedback and sorry for not seeing the `HARDMACRO` mechanism. I updated that change, and I also updated the documentation and bumped the `VERSION`. I did those changes as a separate commit, so if you don't want them, you can just peel them off.

Thanks again for `levee`. I like the idea of an `vi`-like editor small enough that I might actually read all its source.
